src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onAdd: (newTodo: Todo) => void;
+  generateNewId: () => number;
+};
+
+export const TodoForm: React.FC<Props> = ({ onAdd, generateNewId }) => {
+  const [users] = useState(usersFromServer);
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+
+    if (hasTitleError) {
+      setHasTitleError(false);
+    }
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+
+    if (hasUserIdError) {
+      setHasUserIdError(false);
+    }
+  };
+
+  const resetForm = () => {
+    setTitle('');
+    setUserId(0);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    let isError = false;
+
+    if (!title) {
```

Comment:
```suggestion
    if (!title.trim()) {
```
to fix it

https://github.com/mate-academy/react_add-todo-form/assets/70220667/0b833b81-9789-4c70-bac6-4c13a9059456